<ui version="4.0" >
 <class>localetime</class>
 <widget class="QWidget" name="localetime" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_8" >
     <item>
      <widget class="QLabel" name="labCalendarSystem" >
       <property name="text" >
        <string>Calendar system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_comboCalendarSystem" />
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>421</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="m_groupFormat" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="m_labTimeFmt" >
          <property name="text" >
           <string>Time format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboTimeFmt" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="m_labDateFmt" >
          <property name="text" >
           <string>Date format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboDateFmt" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="m_labDateFmtShort" >
          <property name="text" >
           <string>Short date format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboDateFmtShort" />
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="m_chDateMonthNamePossessive" >
        <property name="text" >
         <string>Use declined form of month name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="m_groupWeek" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="labWeekStartDay" >
          <property name="text" >
           <string>First day of the week:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboWeekStartDay" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="labWorkingWeekStartDay" >
          <property name="text" >
           <string>First working day of the week:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboWorkingWeekStartDay" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <widget class="QLabel" name="labWorkingWeekEndDay" >
          <property name="text" >
           <string>Last working day of the week:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboWorkingWeekEndDay" />
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_7" >
        <item>
         <widget class="QLabel" name="labWeekDayOfPray" >
          <property name="text" >
           <string>Day of the week for religious observance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_comboWeekDayOfPray" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>339</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
